website/cue/reference/components/sources/datadog_agent.cue

```diff
@@ -177,16 +177,26 @@ components: sources: datadog_agent: {
 		decompression: {
 			title: "Configuring the Datadog Agent"
```

Comment:
I think we ought to split this into two, providing the old and new config for a while

Reply:
I don't think there are much users for those options out there (yet), so focusing on the newest configuration seems okay for me, anyway the main point here was to remain consistent with https://docs.datadoghq.com/agent/vector_aggregation/#agent-configuration (BTW I'm only allowed to enable automerge, and doc-only PR never trigger soak tests which are blocking the automatic merge 😞 cc @spencergilbert I think you have the right to merge it)